### PR TITLE
UI issue

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/entities/flair-bi/flair-bi.controller.js [hunk 1]
```diff
@@ -471,6 +474,23 @@
             return features.length > 0 ? false : true;
         }
 
+        function getScheduleReport(visualizationid) {
+            schedulerService
+                .getScheduleReport(addPrefix(visualizationid))
+                .then(function (success) {
+                    var report = success.data.report;
+                    if (report) {
+                        de
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/flair-bi.controller.js:483]: what is de here

File: src/main/webapp/app/entities/flair-bi/flair-bi.controller.js [hunk 2]
```diff
@@ -1284,6 +1307,13 @@
             return editMode;
         }
 
+        function hideThreshold(v) {
+            if (v.metadataVisual.name === "KPI") {
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/flair-bi.controller.js:1311]: place KPI in constants file

File: src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js [hunk 3]
```diff
@@ -138,6 +139,40 @@
...
+                        title: "Error"
+                    });
+                });
+
+            schedulerService
+                .getScheduleReport(vm.vizIdPrefix + visualizationid)
+                .then(function (success) {
+                    var report = success.data.report;
+                    if (report) {
+                        var info = { text: 'Visualization configuration changes please reschedule the report.', title: "Updated" }
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/rightnavbar/flair-bi-rightnavbar.controller.js:164]: check the text with @admin-vizcentric venkat

**Pull request merged**